source/gx/tilix/prefeditor/prefdialog.d

```diff
@@ -1577,6 +1577,11 @@ private:
         bh.bind(SETTINGS_STRIP_FIRST_COMMENT_CHAR_ON_PASTE_KEY, cbStrip, "active", GSettingsBindFlags.DEFAULT);
         add(cbStrip);
 
+        //Strip trailing whitespace on paste
+        CheckButton cbStripTrailing = new CheckButton(_("Strip trailing whitespace"));
```

Comment:
Maybe name this "Strip trailing whitespaces and linebreak characters on paste", to make it a bit more clear *when* those whitespaces and linebreaks are removed, as well as to be similar in style to the preceding "Strip first character of paste if comment or variable declaration" option

Reply:
makes sense, I'll update the text